packages/@cdktf/provider-generator/lib/get/generator/skipped-attributes.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) HashiCorp, Inc
+// SPDX-License-Identifier: MPL-2.0
+
+/**
+ * There are large attributes on little used resources that blow up the
+ * size and compile time of the generated providers by a lot.
+ * To mitigate this issue we replace their entire tree of attributes with
+ * a single any attribute. This can still be used, although less conveniently.
+ */
+const SKIPPED_ATTRIBUTES: string[] = [
```

Comment:
Baking these in is a slippery slope. It sends a message of not supporting arbitrary resources. It also introduces breaking changes anytime the list is changed.

Reply:
Agreed on the breaking changes on each change. That's definitely inconvenient.

However, I think it's not really a slippery slope. The singular purpose of this list is to unblock provider generation for providers that have a schema design that won't be usable in any one of our supported languages. In this case, the attributes in this list prevent us from publishing any new versions of the provider. 

I agree that we may be able to come up with a more sophisticated solution to this problem, and we should. However, we don't know what that solution looks like yet and we want to allow users to be able to use newer versions of this provider through CDKTF now. To that end, I think this PR makes a pragmatic tradeoff for this situation.